haros/haros.py

```diff
@@ -317,10 +319,14 @@ def main(argv = None, source_runner = False):
                             level = logging.DEBUG)
     else:
         logging.basicConfig(level = logging.WARNING)
+
     try:
+        if args.dir:
+            os.chdir(args.dir)
```

Comment:
I actually removed it before pushing. Isn't this redundant? My memory is rusty here, but it seems that a child process cannot affect a current directory for the parent process, so changing back to the old directory seems to be just a waste of cycles. Unless, you tell me that the behavior is different outside the unix universe; then I will be happy to reconsider.